layouts/default.vue

```diff
@@ -2,6 +2,7 @@
   <v-app ref="app">
     <backdrop />
     <v-navigation-drawer
+      v-if="$store.state.page.showNavDrawer"
```

Comment:
Why hide the nav drawer?

Reply:
@Camc314 it's a full screen player.

Reply:
@ferferga Could it maybe be tied to fullscreen-ness of the player? Also the drawer already has a `drawer` model and it could be seen as redundant?

Reply:
Maybe even have an event being emitted from the audio controls to play with the drawer model directly?

Reply:
I agree with Thibault, v-model already does this and it's be better to use that instead

Reply:
For future reference, and as it was discussed in Matrix, we're going to leave this as an ``v-if`` so toggling the nav drawer doesn't mess with the user's behaviour. Going the v-model route would require to store another variable that contains whether the user has the navdrawer hidden or not before toggling ot, so we can act accordingly after unhiding it.

If we want to switch everything to v-model at some point, we need to take care of that.